docs/reference/ingest/apis/get-pipeline.asciidoc

```diff
@@ -41,6 +41,12 @@ GET /_ingest/pipeline/my-pipeline-id
 
 `GET /_ingest/pipeline`
 
+[[get-pipeline-api-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have the
+`read_pipeline`, `manage_pipeline`, `manage_ingest_pipelines`, or `manage`
```

Comment:
I'll remove `read_pipeline` from the 7.10 backport. That privilege was added in 7.11.